### DISCUSSION TITLE
Can not install @rainbow-me/rainbowkit

### DISCUSSION THREAD

**Discussion by leonelosz** [rainbow-me/rainbowkit#1965] [Q&A] [answered]
Hi, I've been trying to install @rainbow-me/rainbowkit using the following command

```
npm i @rainbow-me/rainbowkit
```

however installation is stucked forever at 

```
(#######⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂) ⠙ idealTree:ws: sill placeDep ROOT utf-8-validate@6.0.3 OK for: @metamask/sdk-communication-layer@0.18.5 want: ^6.0.3
```

Also Ctrl + C does not work and I had to kill the npm install process.

I'm using "next": "^14.2.3" and node v20.13.0 through nvm

Can you help me? please I really need my project working ASAP.

**Comment by chloe-tan** [comment, marked as answer]
@leonelos-001 

I've been running into this problem too. I believe its related to the recent react `18.3.1` upgrade in the rainbowkit deps, causing certain peer dependencies down the tree to break. 

![image](https://github.com/rainbow-me/rainbowkit/assets/95644202/b17058ef-4276-41e7-874d-ee5efee059e3)

A few ways i bypassed this problem while waiting for the other packages to catch up in versioning:

1. Fix your project's react version to `18.2.0` (meaning not `^18.2.0` or `18.3.1` etc, but the fixed `18.2.0` version), OR
2. Run `npm install --legacy-peer-deps` instead to ignore peer dependency errors

Both methods worked for me, but I'd recommend (1) to remain compliant with peer dependencies :)

**Reply by leonelosz** [reply]
Amazing! Second solution worked for me like a charm! Thank you very much! You saved my day  😄

**Comment by 0xmkh** [comment]
@leonelos-001 This issue should be resolved now. 

Like @chloe-tan said, there are some peer dependencies conflicts during the time of installation. The problem occurs with MetaMask SDK within the wagmi library, which is has some react peer dependencies issues.
